bundles/org.openhab.core.config.core/src/main/java/org/eclipse/smarthome/config/core/i18n/ConfigDescriptionI18nUtil.java

```diff
@@ -40,29 +43,30 @@ public ConfigDescriptionI18nUtil(TranslationProvider i18nProvider) {
         this.i18nProvider = i18nProvider;
     }
 
-    public String getParameterPattern(Bundle bundle, URI configDescriptionURI, String parameterName,
-            String defaultPattern, Locale locale) {
+    public @Nullable String getParameterPattern(@Nullable Bundle bundle, URI configDescriptionURI, String parameterName,
```

Comment:
Same comment like in the `ModuleTypeI18nUtil` regarding the bundle:

```suggestion
    public @Nullable String getParameterPattern(Bundle bundle, URI configDescriptionURI, String parameterName,
```

Reply:
I'll any resolve any null analysis errors this might cause!